ldes-server-domain/src/main/resources/eventstreamShaclShape.ttl

```diff
@@ -29,6 +29,12 @@
     sh:maxCount 1;
     sh:name "versionOfPath"@en;
     sh:path <https://w3id.org/ldes#versionOfPath>
+  ], [
+    sh:nodeKind sh:IRI ;
+    sh:description "The object property that indicates the type of a member."@en;
+    sh:maxCount 1;
```

Comment:
shouldn't we also add minCount 1 here? So that it's a mandatory property (as long as the new member extraction algorithm hasn't been implemented)